ci-operator/config/openshift-kni/commatrix/openshift-kni-commatrix-main.yaml

```diff
@@ -24,6 +36,44 @@ tests:
   commands: make lint
   container:
     from: src
+- as: commatrix-krew-command
+  steps:
+    cluster_profile: aws-telco
+    env:
+      BASE_DOMAIN: telco5g-ci.devcluster.openshift.com
+    test:
+    - as: commatrix-krew-command
+      commands: |
+        # install krew
+        OS="$(uname | tr '[:upper:]' '[:lower:]')"
+        ARCH="$(uname -m | sed -e 's/x86_64/amd64/' -e 's/aarch64/arm64/' -e 's/armv7l/arm/')"
+        KREW="krew-${OS}_${ARCH}"
+        mkdir -p /tmp
+        curl -fsSL "https://github.com/kubernetes-sigs/krew/releases/latest/download/${KREW}.tar.gz" -o /tmp/${KREW}.tar.gz  
+        mkdir -p /tmp/krew
+        tar -xvzf /tmp/${KREW}.tar.gz -C /tmp/krew
+        /tmp/krew/${KREW} install krew
+        export PATH="${KREW_ROOT:-$HOME/.krew}/bin:$PATH"
+        # install the commatrix krew
+        curl -o /tmp/krew/commatrix-krew.yaml https://raw.githubusercontent.com/openshift-kni/commatrix/krew-test/cmd/commatrix-krew.yaml
```

Comment:
Why do you need to copy the file to /tmp? This is running in the repo, can't point directly to the file?